program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js

```diff
@@ -0,0 +1,8 @@
+//inverted pyramid
+for(let a = 6;a>=1;a--){
+    let str =' '
```

Comment:
This code gives an extra double quotes in the last line .
To fix this make the for loop condition as below
`for(let a = 6;a>1;a--){`

Reply:
Ok